src/management-system-v2/components/bpmn-viewer.tsx

```diff
@@ -65,7 +65,7 @@ export const LazyBPMNViewer: FC<LazyLoadingBPMNViewerProps> = ({
   return (
     <>
       <div ref={ViewerContainerRef}>
-        {visible /* This ensures, that only elements, that are visible or close to beeing visible are rendered -> reduces requests for bpmn/xml */ ? (
+        {!visible /* This ensures, that only elements, that are visible or close to beeing visible are rendered -> reduces requests for bpmn/xml */ ? (
```

Comment:
Only to show on preview need to be reverted back to visible before merging

Reply:
Okay, can merge after revert.

Reply:
@MaxiLein don't know if you saw